docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
<p>version controlled platform &lt;- version control platform</p><p> </p><p>How did we pick the list? I think I made a list of other common options in another ticket. University affiliated repositories is also acceptable (e.g. Dataverse etc).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I think I pulled them from page 16 of the Open-Source Science Guidance doc under their "Where to Share Software" section, but I'll look back at your list and add more! They have some other generalist repos also listed on page 12 for where to share data, I think Dataverse is listed there</p>